dom/src/ElementFinder.ts

```diff
@@ -46,3 +39,11 @@ export class ElementFinder {
       .concat(topNodeMatches);
   }
 }
+
+function toElArray(input: any): Array<Element> {
+  return <Array<Element>> Array.prototype.slice.call(input);
+}
+
+interface MatchesSelector {
```

Comment:
Same thing about moving up interfaces and stuff. Everything that is used in line `x` should be defined in line `x - n`